ooniprobe/View/OONIRun/OoniRunViewController.m

```diff
@@ -178,6 +178,7 @@ -(void)validateAndAddURLs{
             }
         } @catch (NSException *exception){
             NSLog(@"OoniRunViewControllerError: %@", exception);
+            [ThirdPartyServices recordError:@"failure"  reason:exception.reason userInfo:exception.userInfo];
```

Comment:
Will the reported exception include the OONI Run link that lead to the failure?

What is `userInfo`? Does this contain any personal information about the user submitting the exception?

Reply:
`userInfo` is a dictionary containing application-specific data pertaining to the receiver.

For more info, see https://developer.apple.com/documentation/foundation/nsexception

Reply:
Can we include also the OONI run link in the exception? I think it's going to be useful to be able to debug issues with it.